### PR TITLE
Add feature to select pixel (sample) aspect ratio

### PR DESCRIPTION
**Pull request by mntone**
This is a workaround for:
https://obsproject.com/mantis/view.php?id=806

But, it uses "par" param.

### CODE REVIEW

File: UI/display-helpers.hpp [hunk 1]
```diff
@@ -43,10 +43,10 @@ static inline void GetScaleAndCenterPos(
 
 static inline void GetCenterPosFromFixedScale(
 		int baseCX, int baseCY, int windowCX, int windowCY,
-		int &x, int &y, float scale)
+		int &x, int &y, float scaleX, float scaleY)
 {
-	x = (float(windowCX) - float(baseCX)*scale) / 2.0f;
-	y = (float(windowCY) - float(baseCY)*scale) / 2.0f;
+	x = (float(windowCX) - float(baseCX)*scaleX) / 2.0f;
```
Comment (RytoEX) [inline UI/display-helpers.hpp]: Add a space on each side of the multiplication operator (asterisk).

File: UI/display-helpers.hpp [hunk 2]
```diff
@@ -43,10 +43,10 @@ static inline void GetScaleAndCenterPos(
...
+	y = (float(windowCY) - float(baseCY)*scaleY) / 2.0f;
```
Comment (RytoEX) [inline UI/display-helpers.hpp]: Add a space on each side of the multiplication operator (asterisk).

**Comment by mntone** [comment]
I resolved the conflicts with the master branch.

**Comment by ONSBalder** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**